src/layout/Panel.jsx

```diff
@@ -117,9 +118,9 @@ const Sidebar = () => {
               setSelected={setSelected}
             />
             <Item
-              title="Roadmap"
-              to="/form"
-              icon={<PersonOutlinedIcon />}
+              title="Analytics"
```

Comment:
Make each component in separate file and reuse here. e.g Item component, Sidebar component

Reply:
seperated